obspy/io/sac/sacpz.py

```diff
@@ -47,7 +47,13 @@ def _write_sacpz(inventory, file_or_file_object):
             for cha in sta:
                 resp = cha.response
                 sens = resp.instrument_sensitivity
-                paz = resp.get_paz()
+                try:
+                    paz = resp.get_paz()
+                except Exception:
+                    print("{}.{}.{} has no paz. Skipping.".format(net.code,
+                                                                  sta.code,
+                                                                  cha.code))
```

Comment:
I would also print the epoch times of the channel to uniquely identify it. And instead of printing please raise warnings.

Reply:
Oh overlooked this.. yes definitely a warning instead.